io.keybase.Keybase.yml

```diff
@@ -1,9 +1,9 @@
 app-id: io.keybase.Keybase
 
 base: org.electronjs.Electron2.BaseApp
-base-version: '22.08'
+base-version: '24.08'
 runtime: org.freedesktop.Platform
-runtime-version: '22.08'
+runtime-version: '24.08'
```

Comment:
This seems to have broken the build, I am getting an error:
```
Emptying app dir 'builddir'
error: org.freedesktop.Sdk/x86_64/24.08 not installed
Failed to init: Unable to find sdk org.freedesktop.Sdk version 24.08
```

I will revert this patch.

Reply:
I figured it out; https://github.com/RalfJung/io.keybase.Keybase/commit/4e6e7e37c7f3995943cac2bfaaf75070cdac907a bumps the version back up again.